static/app/views/replays/deadRageClick/selectorTable.tsx

```diff
@@ -249,6 +249,8 @@ const MessageContainer = styled('div')`
   grid-auto-flow: row;
   gap: ${space(1)};
   justify-items: center;
+  text-align: center;
+  padding: ${space(4)};
```

Comment:
what's this doing?

Reply:
I realized as I was testing this that the style here is a bit off when the screen size is small - i added some padding and text alignment to make it look better

Before:

<img width="509" alt="SCR-20231207-pcbg" src="https://github.com/getsentry/sentry/assets/56095982/6c6baf83-bdbe-4e8a-a501-425bc3cfc877">

After:
<img width="591" alt="SCR-20231207-pcdp" src="https://github.com/getsentry/sentry/assets/56095982/42e11aeb-561f-45b1-b5c2-72dfff6897ff">

Reply:
oof, nice 👍